test/restore.bats

```diff
@@ -12,90 +12,51 @@ function teardown() {
 
 @test "crio restore" {
 	start_crio
-	run crictl runp "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_id="$output"
-
-	run crictl pods --quiet --id "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_list_info="$output"
-
-	run crictl inspectp -o json "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
+	pod_list_info=$(crictl pods --quiet --id "$pod_id")
+
+	output=$(crictl inspectp -o json "$pod_id")
 	pod_status_info=$(echo "$output" | jq ".status.state")
 	pod_ip=$(echo "$output" | jq ".status.ip")
 	pod_created_at=$(echo "$output" | jq ".status.createdAt")
 
-	run crictl create "$pod_id" "$TESTDATA"/container_config.json "$TESTDATA"/sandbox_config.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	ctr_id="$output"
-
-	run crictl ps --quiet --id "$ctr_id" --all
-	echo "$output"
-	[ "$status" -eq 0 ]
-	ctr_list_info="$output"
-
-	run crictl inspect -o table "$ctr_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_config.json "$TESTDATA"/sandbox_config.json)
+	ctr_list_info=$(crictl ps --quiet --id "$ctr_id" --all)
+	output=$(crictl inspect -o table "$ctr_id")
 	ctr_status_info=$(echo "$output" | grep ^State)
 
 	stop_crio
 
 	start_crio
-	run crictl pods --quiet
-	echo "$output"
-	[ "$status" -eq 0 ]
-	[[ "${output}" != "" ]]
+	output=$(crictl pods --quiet)
 	[[ "${output}" == "${pod_id}" ]]
 
-	run crictl pods --quiet --id "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	output=$(crictl pods --quiet --id "$pod_id")
 	[[ "${output}" == "${pod_list_info}" ]]
 
-	run crictl inspectp -o json "$pod_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	output=$(crictl inspectp -o json "$pod_id")
 	status_output=$(echo "$output" | jq ".status.state")
 	ip_output=$(echo "$output" | jq ".status.ip")
 	created_at_output=$(echo "$output" | jq ".status.createdAt")
 	[[ "${status_output}" == "${pod_status_info}" ]]
 	[[ "${ip_output}" == "${pod_ip}" ]]
 	[[ "${created_at_output}" == "${pod_created_at}" ]]
 
-	run crictl ps --quiet --all
-	echo "$output"
-	[ "$status" -eq 0 ]
-	[[ "${output}" != "" ]]
+	output=$(crictl ps --quiet --all)
```

Comment:
check for non-empty output is removed but that's OK because it was redundance since we check for a particular string below.